backend/user_management/routers/profile.py

```
@@ -25,19 +23,21 @@ async def get_profile(user: User = Depends(get_current_user)):
         "phone": user.phone,
     }
 
-# Update Profile
-@router.put("/profile")
-async def update_profile(user_update: UserUpdate, db: Session = Depends(get_db), user: User = Depends(get_current_user)):
+
+@router.patch("/profile")
+async def update_profile(
```

Ruff (0.8.2)

Missing return type annotation for public function `update_profile`

(ANN201)